bsp/boards/openmote-b-24ghz/radio.c

```diff
@@ -45,6 +45,8 @@ typedef struct {
 
 radio_vars_t radio_vars;
 
+// global radio selection, will use RADIOSETTING_24GHZ by default at initialization. 
+uint8_t SELECTED_RADIO      =       RADIOSETTING_24GHZ;
```

Comment:
the name `SELECTED_RADIO` suggests it's a `#define`. Not sure what is going on.

Reply:
fixed it. used to be #define and forgot to change capitalization